Assets/Plugins/Android/Editor/AndroidBuilder.cs

```diff
@@ -36,7 +36,7 @@ namespace PlayEveryWare.EpicOnlineServices.Build
 
     public class AndroidBuilder : PlatformSpecificBuilder
     {
-        public AndroidBuilder() : base("Plugins/Android") { }
+        public AndroidBuilder() : base("Plugins/Android", BuildTarget.Android) { }
```

Comment:
In order for the proper Builder to be determined based on the platform _being built to_, the various builder classes need to be able to indicate for which BuildTargets they work for.

The decentralized nature of this implementation allows for the addition of other platforms without altering anything in the existing build pipeline.